corehq/apps/accounting/subscription_changes.py

```diff
@@ -6,7 +6,7 @@
 from django.utils.translation import ugettext_lazy as _, ungettext
 from corehq import privileges
 from corehq.apps.accounting.utils import get_active_reminders_by_domain_name
-from corehq.apps.app_manager.models import Application
+from corehq.apps.app_manager.models.common import Application
```

Comment:
I still think to neaten up this PR and abstract the fact that the models code is further split up, it's still worth not changing these imports though.